### PR TITLE
Feature/create base formatting

### PR DESCRIPTION
**Pull request by hvohr**
-Added to the html document to include the proper header, main and body elements for the website. 
-The navigation has two elements and a search bar and the main page has three generic recipes displayed (hardcoded currently). 
-Formatted the HTML elements using CSS for proper alignment and styling. 
-Created the recipe test files and corresponding recipe testing + first two functions
-Created the sample data sets for recipe and ingredients

Did this break anything?
- [x] No
- No current warnings or issues

Type of change
- [x] New feature (non-breaking change which adds functionality)

Checklist:
- [x]  The code runs locally
- [x]  The code is DRY. If it's not, I tried my best
- [x]  I reviewed my code before pushing
- What's next: Continuing to test functions that calculate cost, determine ingredient names, and manipulate the DOM
- I did not format the commit messages w/ feat:, and will do so in the future along with other contributors in the group.

### CODE REVIEW

File: src/recipeUtils.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+  return recipes.filter((recipe) => {
+    return recipe.tags.includes(tag)
+  });
+}
+
+export function recipesfromName(recipes, name) {
+  return recipes.filter((recipe) => {
+    return recipe.name.includes(name)
+  })
+}
```
Comment (Denzel-Turambi) [inline src/recipeUtils.js:11]: Single responsibility functions! Very nice!

File: test/RecipeUtils-test.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    const expected = 'Thai Chicken Tenders with Broiled Pineapple Slaw'
+    const filtered = recipesfromName(recipeData, 'Thai Chicken')
+    expect(filtered.length).to.equal(1)
+    expect(filtered[0].name).to.equal(expected)
+  })
+  it ('Should return nothing if no recipes match the user input name', () => {
+    const filtered = recipesfromName(recipeData, 'Spicy Potatoes')
+    expect(filtered.length).to.equal(0)
+  })
+})
```
Comment (Denzel-Turambi) [inline test/RecipeUtils-test.js:38]: Good job on testing happy and sad paths!

File: src/recipeUtils.js (hunk 1 above)
Comment (andreasorensen) [reply src/recipeUtils.js:11]: I love that you ended up using includes! That really helps with the error handling!

File: test/RecipeUtils-test.js (hunk 2 above)
Comment (andreasorensen) [reply test/RecipeUtils-test.js:38]: Great job!

File: dist/index.html
Comment (andreasorensen) [inline dist/index.html:1]: We're still going to remove the total cost from the main page, right?

File: src/data/recipes-sample.js
Comment (andreasorensen) [inline src/data/recipes-sample.js:1]: Awesome job creating these sample sets!

**Review by andreasorensen — APPROVED** [review]
You did a great job with all this Hollis! I think there are a few updates we can make as we go along, but this is a great start! Thanks for all your hard work.

File: test/RecipeUtils-test.js
Comment (fernandorobles97) [inline test/RecipeUtils-test.js:1]: Great work, love how these tests are formatted.

File: test/RecipeUtils-test.js (hunk 2 above)
Comment (fernandorobles97) [reply test/RecipeUtils-test.js:38]: Great work, love how these tests are formatted!

**Review by fernandorobles97 — APPROVED** [review]

**Pull request merged**